README.md

```diff
@@ -73,17 +81,25 @@ TBD
 
 TBD
 
-## Verifying Releases
+## Verifying Integrity
 
-Bomctl releases can be [found here](https://github.com/bomctl/bomctl/releases) and are signed
+### Verifying Container Images
+
+Bomctl container images can be [found here](https://hub.docker.com/repository/docker/bomctl/bomctl/general) and are signed
```

Comment:
This ends up resolving to `https://hub.docker.com/r/bomctl/bomctl` but requires a login. Suggest reordering to allow lowercasing of `bomctl` or just surround with backticks if starting a sentence with it.

```suggestion
Container images for `bomctl` can be found [here](https://hub.docker.com/r/bomctl/bomctl) and are signed
```